apps/src/templates/studioHomepages/CoursesStudentEnglish.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import FindLocalClassBanner from './FindLocalClassBanner';
+import {CourseBlocksHoc} from './CourseBlocks';
+import CourseBlocksGradeBands from './CourseBlocksGradeBands';
+import i18n from "@cdo/locale";
+
+/**
+ * This is the main content for the Courses page for a student using English,
+ * though it may also be shown for a signed-out user using English, and is in
+ * fact the default in such a case.
```

Comment:
Feels overly verbose?

Could just write:
This is the main content for the Courses page for a student using English, as well as the default for a signed-out user using English.

Also, this seems to be in conflict with a comment later in this PR?
`# When signed out, an English user will get the teacher view of /courses.`